rules/config

```diff
@@ -125,6 +125,10 @@ ENABLE_SFLOW = y
 # ENABLE_MGMT_FRAMEWORK - build docker-sonic-mgt-framework for CLI and REST server support
 ENABLE_MGMT_FRAMEWORK = y
 
+# ENABLE_HOST_SERVICE - build sonic-host-services for mgmt-framework and/or
+# telemetry containers to access host functionality
+ENABLE_HOST_SERVICE = n
```

Comment:
What is the significance of this flag "ENABLE_HOST_SERVICE"? Can it be available for all the containers on the sonic? what does it mean y or n?

Reply:
This flag controls whether the host service is built and installed on the host system. This can be made available to any container as long as the D-Bus sockets are mounted within the container (`/var/run/dbus`).

Reply:
fine, thanks